mods/ra/rules/ai.yaml

```diff
@@ -428,21 +424,22 @@ Player:
 		NavalUnitsTypes: ss, msub, dd, ca, lst, pt
 		ConstructionYardTypes: fact
 		NavalProductionTypes: spen, syrd
-		AirUnitsTypes: mig, yak, heli, hind, mh60
```

Comment:
why exclude migs?

Reply:
I find a substantial portion of the damage the air force causes is from planes crashing, this is really just to let it launch more planes faster/cheaper.

Reply:
I find the Allied air units to be more complementary and able to be a bit less likely to be blown up (also less likely to blow up on something important).  I can add migs back at a smaller amount if you want.

Reply:
perhaps just increase the percentage of yaks, I think migs should be fairly important to the AI given it's the anti-armour / anti-allied ship plane